engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -29,6 +29,8 @@ case class BackendCallJobDescriptor(workflowDescriptor: WorkflowDescriptor,
                                     locallyQualifiedInputs: CallInputs = Map.empty,
                                     abortRegistrationFunction: Option[AbortRegistrationFunction] = None) extends JobDescriptor[BackendCallKey] {
 
+  def call = key.scope
```

Comment:
hmm can't think of a great reason for that, maybe `lazy val` them to not force evaluation if callers don't care?